### DISCUSSION TITLE
Can keycloak javascript adapter use OpenIDConnect against Identity Server?

### DISCUSSION THREAD

**Discussion by KnutHovda** [keycloak/keycloak#19078] [Q&A]
I have a Vue/Javascript app that uses the keycloak Javascript adapter to communicate with keycloak, and this works fine. But now we need to deploy the app to another platform where we'll have to use a different identity provider (Identity Server). It supports OIDC, so my question is - is the Javascript adapter able to talk to this provider using OIDC, or do I need to switch to a different library to be able to talk to something else than keycloak?  In advance, thanks for your help!

**Comment by jonkoops** [comment]
Yes, you can construct it with the same parameters you would any other OpenID Connect client. But if you are not integrating with Keycloak I'd recommend using an alternative library.
